main.go

```diff
@@ -54,19 +57,17 @@ func run(ctx context.Context) error {
 		Version,
 	)
 
-	r := mux.NewRouter()
-
-	rend := renderer.New(cfg.RendererURL)
+	router := mux.NewRouter()
 
 	healthcheck := health.New(versionInfo, cfg.HealthCheckCriticalTimeout, cfg.HealthCheckInterval)
-	if err = registerCheckers(ctx, &healthcheck, rend); err != nil {
-		return err
-	}
-	routes.Init(ctx, r, cfg, healthcheck)
+
+	// Initialise render client, routes and initialise localisations bundles
```

Comment:
Instantiating the the renderer library here with frontend configs and combined asset functions (generated by`go-bindata`). To be passed into the routes initialisation